### DISCUSSION TITLE
how to set webp's XMP 'orientation' info

### DISCUSSION THREAD

**Discussion by t00350320** [ImageMagick/ImageMagick#6533] [Help]
### Is your feature request related to a problem? Please describe.

first, we have a jpeg file with exif info, 
then ,we extract the exif orientation info  from jpeg
now, we have a case to transfer a jpeg file to webp.
how can we write this orientation info got from jpeg into webp .
like exiv2 -M "set Xmp.tiff.Orientation 6" 1384382430781112329.webp

### Describe the solution you'd like

we guess if there could be a interface or api function exported, so we can add an xmp data to webp

**Comment by fmw42** [comment]
Why not just correct the orientation when converting using -auto-orient
